documentation/images/images-default.md

```diff
@@ -0,0 +1,58 @@
+⚠️ This documentation file isn't complete ⚠️
+
+It is only there to serve as an example of how we'd document each template
+
+---
+
+## Template Information
+
+| Name      | Description       |
+| --------- | ----------------- |
+| File name | images-default.stencil |
+| Invocation example | `swiftgen storyboard -t default …` |
+| Language | Swift 3 |
+| Author | Olivier Halligon |
+
+## When to use it
+
+This template is the default template used by SwiftGen when parsing images in Asset Catalogs.
+It generates Swift 3 code and is suitable for most needs.
```

Comment:
Yeah that PR was opened before we even decided to get rid of the "default" behavior even. 

Again the idea behind this paragraph, which would be probably more useful for the strings templates especially, is to tell which template to use depending on which banking conventions you have as input (how you name your strings keys for example, are you the kind of people that use all-caps keys, or keys structured with dots or underscore to separate the key parts, etc… and what do you expect as output (structured, not structured…)

Might be less relevant for images template because the naming doesn't impact the structural split (the organisation of the assets catalog does), but you get the idea. Because not everyone has the same naming and organization conventions the aim is to help people choose the right template for them.